rumqttd/src/link/local.rs

```diff
@@ -35,6 +35,98 @@ pub enum LinkError {
     Elapsed(#[from] tokio::time::error::Elapsed),
 }
 
+// used to build LinkTx and LinkRx
+pub struct LinkBuilder<'a> {
+    tenant_id: Option<String>,
+    client_id: &'a str,
```

Comment:
we clone this while creating connection anyway so let's just make it `String`

Reply:
keeping it unchanged! resolving the conversation.